pquery/sort.go

```diff
@@ -48,17 +48,23 @@ func buildTieBreakerFields(dataColumn string, req protoreflect.MessageDescriptor
 	tieBreakerFields := make([]sortSpec, 0, len(fallback))
 	for _, tieBreaker := range fallback {
 
-		if tieBreaker.ColumnName != dataColumn {
-			if len(tieBreaker.Path) > 0 {
-				return nil, fmt.Errorf("tiebreaker is a jsonb-sub-field of an unknown column '%s', with path %#v", tieBreaker.ColumnName, tieBreaker.Path)
+		if tieBreaker.columnName != dataColumn {
+			if len(tieBreaker.pathInColumn) > 0 {
+				return nil, fmt.Errorf("tiebreaker is a jsonb-sub-field of an unknown column '%s', with path %#v", tieBreaker.columnName, tieBreaker.pathInColumn)
 			}
+			tieBreakerFields = append(tieBreakerFields, sortSpec{
+				NestedField: &pgstore.NestedField{
+					RootColumn: tieBreaker.columnName,
+				},
+				desc: false,
+			})
```

Comment:
Without this addition, fallbacks for columns other than the 'data column' (the root holding the JSONB object) are entirely ignored